City-of-Helsinki/drupal-helfi-kymp
# UHF-3252: OpenShift database sync

To test this:

- `make stop`
- `docker compose build`
- `make fresh`

Make sure latest dump is synced from testing and images are served via stage_file_proxy module
